src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpIntelliSense.cs

```diff
@@ -250,8 +250,7 @@ void Main(string[] args)
 assertCaretPosition: true);
         }
 
-        // 🐛 This should work with async completion, but currently does not.
-        [ConditionalWpfFact(typeof(LegacyCompletionCondition)), Trait(Traits.Feature, Traits.Features.Completion)]
+        [WpfFact, Trait(Traits.Feature, Traits.Features.Completion)]
```

Comment:
>WpfFact [](start = 9, length = 7)

I tried this scenario in the recent VS (16.0 preview 4). It work without the fix.

Reply:
Sorry! It seems that my VS worked with the old completion. I'm able to repro the scenario.

Still think we need to make it on the editor side.

---
In reply to: [262195984](https://github.com/dotnet/roslyn/pull/33828#discussion_r262195984) [](ancestors = 262195984)